catalog/app/containers/Search/Results.tsx

```diff
@@ -112,7 +112,7 @@ const useEmptyResultsStyles = M.makeStyles((t) => ({
 interface EmptyResultsProps {
   className?: string
   clearTitle?: string
-  description?: string
+  description?: string | React.ReactNode
```

Comment:
I'd rather use `children` prop for free-form react content. That way you can get rid of React.Fragments

Reply:
yeah true, but i don't want to refactor all the instances of this component